proposals/lip-0047.md

```diff
@@ -172,7 +172,7 @@ This substore contains the snapshot of the active authorities for the current ro
 * The substore prefix is set to `SUBSTORE_PREFIX_SNAPSHOT`.
 * Each substore key is `uint32be(roundNumber)`, where `roundNumber` can be 0, 1 or 2 corresponding to the current round, the next round and in two rounds respectively.
 * Each substore value is the serialization of an object following `snapshotStoreSchema`.
-* Notation: Let `snapshotStore(roundNumber)` be the substore entry with prefix `SUBSTORE_PREFIX_SNAPSHOT` and key `uint32be(roundNumber)`.
+* Notation: Let `snapshotStore(roundNumber)` be the substore entry with prefix `SUBSTORE_PREFIX_SNAPSHOT`, deserialized using the `snapshotStoreSchema` schema,  and key `uint32be(roundNumber)`.
```

Comment:
To be consistent with `Notation` part for other substores:
```suggestion
* Notation: Let `snapshotStore(roundNumber)` be the substore entry in the snapshot substore with the key `uint32be(roundNumber)`, deserialized using the `snapshotStoreSchema` schema.
```

Reply:
Done